app/models/conclusion_final_review.rb

```diff
@@ -110,8 +110,14 @@ def duplicate_review_findings
         end
 
         if Current.global_weakness_code && finding.kind_of?(Weakness)
-          last_code                 = last_code.next
-          final_finding.review_code = finding.review_code = last_code
+          if finding.repeated_of.present?
+            code = finding.repeated_of.review_code
+          else
+            last_code = last_code.next
+            code = last_code
```

Comment:
```suggestion
            code = last_code = last_code.next
```
Son un toque raras las asignaciones así, pero acá no queda _tan_ complicada de entender.

Reply:
me gusta me gusta